src/server/kiwix-serve.cpp

```diff
@@ -879,26 +879,46 @@ int main(int argc, char **argv) {
   }
 
   /* Compute the Welcome HTML */
-  string welcomeBooksHtml;
+  string welcomeBooksCss = ""
+"<style>"
+".book__list { text-align: center; }"
+".book {"
+    "display: inline-block; vertical-align: bottom; margin: 10px; padding: 15px 20px; width: 300px;"
+    "border: 1px solid #ccc; border-radius: 15px;"
+    "text-align: left; color: #000; font-family: sans-serif; font-size: 13px;"
+"}"
+".book:hover { background-color: #eee; box-shadow: 2px 2px 5px 0px #ccc}"
+".book__background { background-repeat: no-repeat; background-size: auto; background-position: top right; }"
+".book__title { font-size: 18px; color: #0645ad; }"
+".book__description { padding: 5px 50px 5px 0px; font-size: 15px; }"
+".book__info { line-height: 18px; color: #777; font-weight: bold; font-size: 13px; }"
+"</style>";
```

Comment:
Just to keep the html and it's style at the same place, it's easier to code and debugg.
It's not a bad idea to put it in the template file though